src/Rsk.DuendeIdentityServer.AuditEventSink/Adapters/TokenRevokedSuccessEventAdapter.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Duende.IdentityServer.Events;
+using RSK.Audit;
+
+namespace Rsk.DuendeIdentityServer.AuditEventSink.Adapters
+{
+    public class TokenRevokedSuccessEventAdapter : IAuditEventArguments
+    {
+        private readonly TokenRevokedSuccessEvent evt;
+
+        public TokenRevokedSuccessEventAdapter(TokenRevokedSuccessEvent evt)
+        {
+            this.evt = evt ?? throw new ArgumentNullException(nameof(evt));
+        }
+
+        public ResourceActor Actor => new ResourceActor(ResourceActor.UserSubjectType, evt.ClientId, evt.ClientName);
+        public string Action => evt.Name;
+        public AuditableResource Resource => new AuditableResource("IdentityServer", evt.ClientId);
```

Comment:
The event has a property called `TokenType`, we can set the AuditableResource Identifier to this